lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -314,9 +314,16 @@ void Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
       auto wDims = flattenCdr(w->dims(), axis_w);
```

Comment:
@beicy : I think this piece of code can be simpler:
```
Tensor tmp;
if (w->dims().size() > 2) {
  auto wDims = flattenCdr(w->dims(), axis_w);
  tmp.reset(ElemKind::FloatTy, {wDims.first, wDims.second});
  tmp.copyRawFrom(w);
  w = &tmp;
}
Tensor wtag;
if (typeName == "FC") {
  w->transpose(&wtag, {1, 0});
} else {
  wtag.assign(w);
}
```

Reply:
@artemrakhov Yes, it looks much better :) Thanks a lot!